Sherlock/replicator.h

```diff
@@ -171,39 +170,39 @@ class SubscribableRemoteStream final {
       const auto idxts = ParseJSON<idxts_t>(split[0]);
       assert(idxts.index == index_);
       auto entry = ParseJSON<TYPE_SUBSCRIBED_TO>(split[1]);
-      const idxts_t unused_idxts;
       ++index_;
-      if (subscriber_(std::move(entry), idxts, unused_idxts) == ss::EntryResponse::Done) {
+      if (subscriber_(std::move(entry), idxts, unused_idxts_) == ss::EntryResponse::Done) {
```

Comment:
Note to self: `unused_idxts_` is very dangerous concept that can cause a lot of troubles if subscriber relies on the value passed here to be valid. I.e. interfaces/behavior of universal subscribers that can be attached to the local and remote streams must be revised.
CC @dkorolev